ClientStatus.cs

```diff
@@ -389,6 +397,8 @@ public struct MiscellaneousStruct
             public float? TIME;
             [ProtoMember(24)]
             public string TOOLTIPTEXT;
+            [ProtoMember(25)]
+            public HOTKEYStruct[] HOTKEYS;
```

Comment:
Do you mind keeping this sorted, i.e. HOTKEYS should go up where the H fits, and protomember indexes updated ?